lib/video_info/providers/youtubeplaylist_scraper.rb

```diff
@@ -0,0 +1,59 @@
+require 'oga'
+require 'open-uri'
+require 'open_uri_redirections'
+
+class VideoInfo
+  module Providers
+    module YoutubePlaylistScraper
+      def description
+	meta_nodes = data.css('meta')
+
+	description_node = meta_nodes.detect do 
+	  |m| m.attr('name').value == 'description'
+	end
+
+	description_node.attr('content').value
+      end
+      
+      def title
+	meta_nodes = data.css('meta')
+	
+	title_node = meta_nodes.detect do
+	  |m| m.attr('name').value == 'title'
+	end
+	
+	title_node.attr('content').value
+      end
+      
+      def videos
+	raise(NotImplementedError, 'To access videos, you must provide an API key to VideoInfo.provider_api_keys')
```

Comment:
Line is too long. [107/80]<br>Use 2 (not -5) spaces for indentation.<br>Tab detected.